web/app/static/tags/stack/configurehostgroups.tag

```diff
@@ -64,7 +78,7 @@
                             <div class="form-group">
                                 <label class="col-sm-3 control-label" for="securityGroupNameName{{$index}}">{{msg.cluster_form_securitygroup_label}}</label>
                                 <div class="col-sm-8">
-                                    <select class="form-control" id="securityGroupNameName{{$index}}" name="securityGroupNameName{{$index}}" ng-model="instanceGroup.securityGroupId" ng-options="securitygroup.id as securitygroup.name for securitygroup in $root.securitygroups | orderBy:'name'" ng-required="activeCredential">
+                                    <select class="form-control" id="securityGroupNameName{{$index}}" name="securityGroupNameName{{$index}}" ng-model="instanceGroup.securityGroupId" ng-options="securitygroup.id as securitygroup.name for securitygroup in $root.securitygroups | filter: {'cloudPlatform': activeCredential.cloudPlatform} | orderBy:'name'" ng-required="activeCredential">
```

Comment:
I totally forgot shell commands thanks for pointing that, i will fix shell command.
On default security group, Initially i thought its been done through sql script (updated cloudplatform for existing security group in sql script), but actually thats happening in java call ( get apis). I will update the default security group creation method to add default groups for all platform.
UI changes are already incorporated.